layout-server/src/test/java/org/finos/vuu/layoutserver/service/LayoutServiceTest.java

```diff
@@ -22,9 +12,19 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.dao.EmptyResultDataAccessException;
 
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.*;
```

Comment:
I would revert the star imports and put all static imports at the top (this applies to all files)

Reply:
[Peter recommended to use "organise imports"](https://github.com/ScottLogic/finos-vuu/pull/92/files/043fed7f33254a4215dc3ac83d55f1d155148f29#r1378693730) in IntelliJ. That is what is applying the change to the imports here

Reply:
are you using some other standard for sorting/optimising imports? I am not opinionated on this topic so I am happy either way but we should make sure that we are all using the same standards